bitnami/kafka/templates/_helpers.tpl

```diff
@@ -252,6 +252,13 @@ Return the Kafka configuration configmap
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the Kafka client configuration configmap
+*/}}
+{{- define "kafka.client.configmapName" -}}
+    {{- printf "%s-client-configuration" (include "common.names.fullname" .) -}}
+{{- end -}}
+
```

Comment:
Helper used to choose between the passwordsSecretName or the secret generated using user-provided plaintext passwords.
```suggestion

{{/*
Returns the secret name for the Kafka Provisioning client
*/}}
{{- define "kafka.client.passwordsSecretName" -}}
{{- if .Values.provisioning.auth.tls.passwordsSecret -}}
    {{- printf "%s" (tpl .Values.provisioning.auth.tls.passwordsSecret $) -}}
{{- else -}}
    {{- printf "%s-client-secret" (include "common.names.fullname" .) -}}
{{- end -}}
{{- end -}}

```